tests/scripts/booking/cancel_expired_bookings_test.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+from datetime import timedelta
+
+import pytest
+from pytest import approx
+
+from pcapi.core.bookings.factories import BookingFactory
+from pcapi.core.bookings.models import BookingCancellationReasons
+from pcapi.core.offers.factories import ProductFactory
+from pcapi.models import offer_type
+from pcapi.repository import repository
+from pcapi.scripts.booking.cancel_expired_bookings import cancel_expired_bookings
+
+
+@pytest.mark.usefixtures("db_session")
+class CancelExpiredBookingsTest:
+    def test_old_non_subscription_thing_booking(self, app) -> None:
```

Comment:
c'est typiquement le genre de tests sur lesquels il serait intéressant de voir la règle métier ressortir
```suggestion
    def should_cancel_old_non_subscription_thing_booking(self, app) -> None:
```